packages/gdb/build.sh

```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://www.gnu.org/software/gdb/
 TERMUX_PKG_DESCRIPTION="The standard GNU Debugger that runs on many Unix-like systems and works for many programming languages"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=9.2
-TERMUX_PKG_REVISION=3
+TERMUX_PKG_VERSION=10.1
+TERMUX_PKG_REVISION=1
```

Comment:
TERMUX_PKG_REVISION can be removed completely when version is increased (REVISION is set to 0 per default)

Reply:
oops. Thanks for pointing out!